[2025-01-01T08:24:33.127+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-01T08:24:33.173+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: model_prediction_dag.call_model_api_task manual__2025-01-01T08:14:15.813656+00:00 [queued]>
[2025-01-01T08:24:33.206+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: model_prediction_dag.call_model_api_task manual__2025-01-01T08:14:15.813656+00:00 [queued]>
[2025-01-01T08:24:33.211+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 4
[2025-01-01T08:24:33.331+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): call_model_api_task> on 2025-01-01 08:14:15.813656+00:00
[2025-01-01T08:24:33.350+0000] {standard_task_runner.py:72} INFO - Started process 536 to run task
[2025-01-01T08:24:33.355+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'model_prediction_dag', 'call_model_api_task', 'manual__2025-01-01T08:14:15.813656+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/travel_prediction.py', '--cfg-path', '/tmp/tmpyo09n0ob']
[2025-01-01T08:24:33.363+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask call_model_api_task
[2025-01-01T08:24:33.588+0000] {task_command.py:467} INFO - Running <TaskInstance: model_prediction_dag.call_model_api_task manual__2025-01-01T08:14:15.813656+00:00 [running]> on host 23f622abfe56
[2025-01-01T08:24:33.760+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='model_prediction_dag' AIRFLOW_CTX_TASK_ID='call_model_api_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T08:14:15.813656+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-01T08:14:15.813656+00:00'
[2025-01-01T08:24:33.761+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-01T08:24:33.843+0000] {logging_mixin.py:190} INFO - Prediction: {'prediction': [630.4428]}
[2025-01-01T08:24:33.844+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-01T08:24:33.864+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-01T08:24:33.865+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=model_prediction_dag, task_id=call_model_api_task, run_id=manual__2025-01-01T08:14:15.813656+00:00, execution_date=20250101T081415, start_date=20250101T082433, end_date=20250101T082433
[2025-01-01T08:24:33.941+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-01T08:24:33.973+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-01T08:24:33.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
